### ISSUE TITLE
GRPO training from official SFT checkpoint: reward always 0, all_wrong=1.0

### ISSUE THREAD

**Issue by Heisenberg10110** [tulerfeng/Video-R1#86]
Hello, thank you for releasing the code and models.
I tried to run GRPO training starting from the official SFT checkpoint you provided. However, during training I consistently observe the following log outputs:
```
{'loss': 0.0,
 'grad_norm': 1.9272357593331177,
 'learning_rate': 9.79e-07,
 'zero_var_ratio': 1.0,
 'completion_length': 748.11,
 'rewards/accuracy_reward': 0.0,
 'rewards/format_reward': 0.0,
 'all_wrong': 1.0,
 'all_correct': 0.0,
 'temporal_rewards': 0.9167,
 'reward': 0.0,
 'reward_std': 0.0,
 'epoch': 0.09}

```
